ci/test/pipeline.template.yml

```
@@ -141,6 +141,14 @@ steps:
           queue: linux-x86_64
         coverage: skip
 
+      - id: vet-check
+        label: Cargo vet Check
+        command: bin/ci-builder run stable cargo vet check
```

```suggestion
        command: bin/ci-builder run stable cargo vet check --locked
```

Passing `--locked` will ensure that `cargo vet` in CI doesn't go updating any of the imported audit lists.